fmrest/server.py

```diff
@@ -413,6 +418,9 @@ def get_record(self, record_id: int, portals: Optional[List[Dict]] = None,
 
         params = build_portal_params(portals, True) if portals else {}
 
+        # Date format: 0-US (MM/DD/YYYY, default), 1-file locale, 2-ISO8601 (YYYY-MM-DD)
+        params['dateformats'] = dateformats if dateformats in API_DATEFORMATS else '0'
```

Comment:
I also think it would make more sense to have the ISO format by default, but Claris has chosen the 0-US format.
Is it a good idea to change the default behavior of FMS?
And it may indeed be a little late to change the default value for those already using the library.